.github/ISSUE_TEMPLATE/crash-report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Crash report
+about: Report a game crash.
+title: My game crashed
+labels: Crash
+assignees: ''
+
+---
+
+<!-- This is a guideline that shall help you to include all the required information we depend on to investigate and fix game-breaking bugs. Please follow the instructions and replace any placeholders that are written in capital letters. Instructions like this comment will not be visible in your report. -->
+
+## System Information
+<!-- Information about the operating system, engine version, game mod and package source are mandatory for investigating and fixing crashes. -->
+
+- Operating System:  OPERATING SYSTEM
+- OpenRA Version: ENGINE VERSION
+- OpenRA Mod: GAME MOD
+- Source: Official download package OR self-compiled OR third-party package
+- For self-compiled or third-party packages: Mono version
+
+
+## Exception log
+<!-- Please replace the placeholder below with the content of the exception.log file. The three backsticks before and after the placeholder are used for formatting, so don't remove them. If you don't find the log folder consult https://github.com/OpenRA/OpenRA/wiki/FAQ#my-game-just-crashed. -->
+
+```
+PASTE LOG HERE
+```
+
+
+## Replay
+<!-- Please compress and attach the replay file when OpenRA crashed during a running game. Providing the replay is a great help for us to investigate the issue! Please add a comment at what time in the replay the game crashes. -->
```

Comment:
IMO we should back off on this request a bit, in case it puts people off filing a report. How about something like `<!-- If you have a replay file for the game that crashed, and it crashes again when you play it back, it will be a great help for us t fix the issue. Please compress the replay into a zip file and drag it here to include in the report -->`?